packages/sensors/test/index.test.ts

```
@@ -0,0 +1,675 @@
+                mockBattery.dispatchEvent(new Event("levelchange"));
+              } else if (count === 2) {
+                expect(val.level).toBe(0.5);
+                dispose();
+                resolve();
+              }
+            },
+          );
+
+          await Promise.resolve();
+          setTimeout(() => reject(new Error("timed out")), 500);
+        }),
```

Biome (2.4.15)

[error] Promise executor functions should not be `async`.

(lint/suspicious/noAsyncPromiseExecutor)